analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -47,7 +47,7 @@ internal struct SolutionBuilder
             "System.Linq",
             "System.Xml.Linq",
             "System.Threading.Tasks"
-        };
+        }.Select(GlobalImport.Parse).ToArray();
```

Comment:
This creates an array to parse it and then copy the result to another array. What do you think about using the overload which takes a list of strings: https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.visualbasic.globalimport.parse?view=roslyn-dotnet-3.10.0#Microsoft_CodeAnalysis_VisualBasic_GlobalImport_Parse_System_Collections_Generic_IEnumerable_System_String__

Reply:
Waaay better :)